examples/2d/move_sprite.rs

```diff
@@ -1,46 +1,54 @@
-//! Renders a 2D scene containing a single, moving sprite.
+//! Renders a 2D scene in which the *Bevy* logo is rendered as a sprite, moving up and down.
 
 use bevy::prelude::*;
 
 fn main() {
     App::new()
         .add_plugins(DefaultPlugins)
         .add_startup_system(setup)
-        .add_system(sprite_movement)
+        .add_system(update_moveables)
+        .add_system(update_birds)
```

Comment:
The system order matters here. An inconsistent ordering will result in stuttering and surprising delay.

```suggestion
        .add_system(update_moveables)
        .add_system(update_birds.before(update_moveables))
```

Reply:
Perhaps also add a comment reflecting that?

Reply:
I thought about that but chose to leave out the ordering because I could not decide that tutorialising sytem dependencies was in-scope for the example. But, since you suggested the change, I now have an argument to do so.

Reply:
I wouldn't necessarily "tutorialize system dependencies" but a brief comment explaining why this particular ordering is important would be nice.